packages/devtools-source-map/src/index.js

```diff
@@ -8,57 +7,19 @@ const {
   isOriginalId,
 } = require("./util");
 
-const { workerUtils: { workerTask }} = require("devtools-modules");
-const { setConfig, getValue } = require("devtools-config");
+const { workerUtils: { WorkerDispatcher }} = require("devtools-utils");
 
 import type { Location } from "debugger-html";
 
-// TODO: Rename this to something not specific to debugger (#265)
-// $FlowIgnore: global DebuggerConfig
-setConfig(DebuggerConfig);
+const dispatcher = new WorkerDispatcher();
 
-let sourceMapWorker;
-function restartWorker() {
-  if (sourceMapWorker) {
-    sourceMapWorker.terminate();
-  }
-
-  sourceMapWorker = new Worker(getValue("workers.sourceMapURL"));
-  sourceMapWorker.onerror = () => {
-    console.error("Error in source map worker");
-  };
-}
-
-restartWorker();
-
-function destroyWorker() {
-  if (sourceMapWorker) {
-    sourceMapWorker.terminate();
-    sourceMapWorker = null;
-  }
-}
-
-async function hasMappedSource(location: Location): Promise<boolean> {
```

Comment:
I moved `hasMappedSource` into `./source-map` because it went over the wire anyways and was more logically grouped.